.github/workflows/release.yml

```diff
@@ -2,33 +2,27 @@ name: Release
 
 on:
   push:
-    branches:
-      - master
     tags:
       - "v*"
 
 jobs:
   build:
+    if: ${{ github.head_ref }} == "master"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Exit if not on master branch
```

Comment:
this doubly doesn't work:

problem #1: `github.ref` is the tag ref, not the branch
problem #2: `exit 0`  doesn't exit the action, just the run block